convert_jeff32.py

```diff
@@ -47,59 +42,84 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
+if args.destination is None:
+    args.destination = Path('-'.join([library_name, args.release, 'hdf5']))
+
+# This dictionary contains all the unique information about each release. This can be exstened to accommodated new releases
+release_details = {
+    '3.2':{
+        'base_url': 'https://www.oecd-nea.org/dbforms/data/eva/evatapes/jeff_32/Processed/',
+        'files':['JEFF32-ACE-'+temperature+'K.tar.gz' for temperature in args.temperatures]
+                +['TSLs.tar.gz'],
+        'neutron_files':ace_files_dir.rglob('*.ACE'),
+        'metastables': ace_files_dir.rglob('*M.ACE'),
+        'sab_files': ace_files_dir.glob('ANNEX_6_3_STLs/*/*.ace'),
+        'redundant': ace_files_dir.glob('ACEs_293K/*-293.ACE'),
+        'compressed_file_size': '9 GB',
+        'uncompressed_file_size': '40 GB'
+    }
+}
+
+download_warning = """
+WARNING: This script will download up to {} GB of data. Extracting and
+processing the data may require as much as {} GB of additional free disk
+space. Note that if you don't need all 11 temperatures, you can modify the
+'files' list in the script to download only the data you want.
+""".format(release_details[args.release]['compressed_file_size'],
+           release_details[args.release]['uncompressed_file_size'])
 
-base_url = 'https://www.oecd-nea.org/dbforms/data/eva/evatapes/jeff_32/Processed/'
-files = ['JEFF32-ACE-293K.tar.gz',
-         'JEFF32-ACE-400K.tar.gz',
-         'JEFF32-ACE-500K.tar.gz',
-         'JEFF32-ACE-600K.tar.gz',
-         'JEFF32-ACE-700K.tar.gz',
-         'JEFF32-ACE-800K.zip',
-         'JEFF32-ACE-900K.tar.gz',
-         'JEFF32-ACE-1000K.tar.gz',
-         'JEFF32-ACE-1200K.tar.gz',
-         'JEFF32-ACE-1500K.tar.gz',
-         'JEFF32-ACE-1800K.tar.gz',
-         'TSLs.tar.gz']
+print(download_warning)
 
 # ==============================================================================
 # DOWNLOAD FILES FROM OECD SITE
 
 if args.download:
-    for f in files:
-        download(urljoin(base_url, f))
+    for f in release_details[args.release]['files']:
+        download(urljoin(release_details[args.release]['base_url'], f))
 
 # ==============================================================================
 # EXTRACT FILES FROM TGZ
 
 if args.extract:
-    for f in files:
+    for f in release_details[args.release]['files']:
         # Extract files
         if f.endswith('.zip'):
             with zipfile.ZipFile(f, 'r') as zipf:
                 print('Extracting {}...'.format(f))
-                zipf.extractall('jeff-3.2')
+                zipf.extractall(ace_files_dir)
 
         else:
             suffix = 'ACEs_293K' if '293' in f else ''
             with tarfile.open(f, 'r') as tgz:
                 print('Extracting {}...'.format(f))
-                tgz.extractall(os.path.join('jeff-3.2', suffix))
+                tgz.extractall(ace_files_dir.joinpath(suffix))
```

Comment:
Unless you have multiple path components your appending, I find the `/` operator more visually appealing
```suggestion
                tgz.extractall(ace_files_dir / suffix)
```

Reply:
Ok I shall use / from now on